### DISCUSSION TITLE
Vercel depoyment fails after adding Tests

### DISCUSSION THREAD

**Discussion by augustinanders** [neuefische/questions#592] [Web] [answered]
Hello there,

I have encountered an issue with my project's deployment on Vercel after adding some tests. The tests seem to run fine locally, and `localhost` is also working without any problems. However, when I push the project to GitHub and try to deploy it on Vercel, the deployment fails. I ran `npm run build` locally, but it did not give me any errors. Despite commenting out every test, the problem persists.

Here are the error messages I am getting:

```
19:28:02.824 | Error: Command "npm run build" exited with 1
19:28:03.618 | Deployment completed
19:28:03.153 | BUILD_UTILS_SPAWN_1: Command "npm run build" exited with 1
```

You can find the pull request containing my changes here: [link to PR](https://github.com/augustinanders/2023_03_01_capstone-neuefische_memory/pull/47).

Can you help me figure out what might be causing the deployment to fail?

Thank you for your Help 🙏.

**Comment by onemanwenttomow** [comment, marked as answer]
Hello, 

I think the issue is the location of this [`pages/highscores/highscores.styled.js`](https://github.com/augustinanders/2023_03_01_capstone-neuefische_memory/pull/47/files#diff-343c9482bfc98df6e1f4d06f949ed0afd31e7a7096386b7f42453acf01039eeb). At the moment it is located in the `pages` directory, but it needs to be in your `components` directory. If you expand the error message on vercel you will probably see something complaining about React components inside the pages directory needing to be export default.

**Reply by onemanwenttomow** [reply]
Here are a couple of screenshots that hopefully help. 
<img width="1241" alt="expand" src="https://user-images.githubusercontent.com/1374487/237018162-d1f8e25a-81c7-4d16-9d42-065051727774.png">

Once expanded you can scroll through the build log looking for the issue. 

<img width="1222" alt="details" src="https://user-images.githubusercontent.com/1374487/237018152-581ff648-a50c-480c-8e1e-8e7dbe63cb48.png">

**Reply by augustinanders** [reply]
thanks! this worked out!🚀
